src/plots/cartesian/graph_interact.js

```diff
@@ -629,7 +629,7 @@ function hover(gd, evt, subplot) {
     }
 
     // don't emit events if called manually
-    if(evt.target && !hoverChanged(gd, evt, oldhoverdata)) return;
+    if(!evt.target || !hoverChanged(gd, evt, oldhoverdata)) return;
```

Comment:
now it did ;) https://github.com/plotly/plotly.js/pull/1461/commits/052417bb27cdcaffe449b47bac8f9b7d9a81161f

I'll note in case this confuses someone later: a bunch of the other tests in the `hover_label_test` suite look like they're providing pixel positions but are actually hitting [this block](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/graph_interact.js#L419) that takes the middle of the x axis (and equivalently for y just below)